next.config.js

```diff
@@ -77,6 +77,7 @@ const nextConfig = {
       },
     ];
   },
+  devIndicators: false,
```

Comment:
This stops the Next.js dev tools icon from showing in the corner of the screen - I for one would rather it wasn't there.